src/packages/toast/toast.scss

```
@@ -98,7 +98,9 @@
     /* #ifdef harmony dynamic*/
     line-height: normal;
     /* #endif */
+    /* #ifndef harmony dynamic*/
```

Stylelint (17.11.1)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)